cv32e40p/tests/cfg/pulp_fpu_1cyclat.yaml

```diff
@@ -0,0 +1,15 @@
+name: pulp_fpu_1cyclat
+description: Sets PULP instructions and FPU parameters to 1
+compile_flags: 
+    +define+PULP +define+FPU +define+FPU +define+FPU_ADDMUL_LAT=1 +define+FPU_OTHERS_LAT=1
```

Comment:
Minor comment for readability:
1. `FPU` is defined twice.
2. This could also be written as `+define+PULP+FPU+FPU_ADDMUL_LAT=1+FPU_OTHERS_LAT=1`